deploy/k8s/chart/templates/guac/graphql/030-Deployment.yaml

```diff
@@ -37,14 +37,67 @@ spec:
           imagePullPolicy: {{ .Values.imagePullPolicy }}
           name: service
           command: ["/opt/guac/guacgql"]
-          args: []
+          args:
+            - "--gql-backend"
+            - "ent"
+            - "--db-address"
+            - "postgres://$(PGUSER)@$(PGHOST):$(PGPORT)/$(DB_NAME)"
+            - "--db-driver"
+            - "postgres"
+            - "--db-debug"
+            - "true"
+            - "--gql-debug"
+            - "true"
           workingDir: "/guac"
           env:
             - name: NAMESPACE
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
             {{ include "trustification.http-server" . | nindent 12 }}
+            - name: PGHOST
+              valueFrom:
+                secretKeyRef:
+                  name: guac-admin-db
```

Comment:
Please to not use the `admin` connection. That should be the `user` connection.

Reply:
Fixed